packages/cli/src/metadataGeneration/transformer/transformer.ts

```diff
@@ -0,0 +1,51 @@
+import type { HasModifiers } from 'typescript';
+import { SyntaxKind, getModifiers } from 'typescript';
```

Comment:
```suggestion
import { SyntaxKind, getModifiers, type HasModifiers } from 'typescript';
```

Weird that Eslint doesn't handle this.